pkg/services/alerting/rule.go

```diff
@@ -72,20 +88,22 @@ func (e ValidationError) Error() string {
 	return fmt.Sprintf("alert validation error: %s", extraInfo)
 }
 
-var (
-	valueFormatRegex = regexp.MustCompile(`^\d+`)
-	unitFormatRegex  = regexp.MustCompile(`\w{1}$`)
-)
+func getTimeDurationStringToSeconds(str string) (int64, error) {
+	if str == "0" || str == "" {
```

Comment:
Frequency shouldn't allow an empty or zero input. Sorry if I wasn't clear on that.